build.sh

```diff
@@ -71,7 +71,7 @@ build_server(){
     echo "copying requied files to app_dist"
     cp -R libs helpers proxy resourcebundles package.json framework.config.js sunbird-plugins routes constants controllers server.js ./../../Dockerfile app_dist
     cd app_dist
-    nvm use 16.0.0
+    nvm use $NODE_server
```

Comment:
make it **NODE_VERSION** instead of NODE_server

Reply:
Changed to NODE_VERSION